ZupIT/horusec
# Develop

* Removing some unnecessary paths ignored by horusec (#502)
* Removing duplicated issues (#501)
* Feature/output txt (#496)
* Builds binaries for platforms and draft a GitHub release (#503)
* Move tfsec formatter to conform to architectural standards (#508)
* Feature/trivy (#504)
* Tests: refactor end-to-end tests (#505)
* Javascript rules improve (#506)
* Remove unused constant (#511)
* Tests: fix expected vulnerabities on Javascript (#512)
* Add Checkov as HCL analyzer (#510)
* Normalize interfaces to follow Go standards (#509)
